dashboard/src/App.tsx

```
@@ -33,177 +68,472 @@ export default function App() {
+            >
+              <span className="flex items-center gap-2">
+                <BookOpen className="h-4 w-4" strokeWidth={2.5} />
+                Docs
+              </span>
+            </button>
+          </nav>
+
+          <a
+            href="https://github.com/rohitg00/manim-video-generator"
+            target="_blank"
+            className="p-2 border-2 border-[#2d2d2d] bg-white shadow-hard-sm hover:shadow-none hover:translate-x-[2px] hover:translate-y-[2px] transition-all"
```

Biome (2.3.13)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)